hybrasyl/Messaging/Statuses.cs

```diff
@@ -21,7 +21,7 @@ public new static ChatCommandResult Run(User user, params string[] args)
         {
             if (Game.World.WorldData.TryGetValueByIndex(args[0], out Status status))
             {
-                user.ApplyStatus(new CreatureStatus(status, user, null));
+                user.ApplyStatus(new CreatureStatus(status, user, null, null));
```

Comment:
This should be corrected, as "source" does not need to exist.

Reply:
See above.

Reply:
See above as well.